src/admin/client.py

```
@@ -0,0 +1,186 @@
+            params={"dir": direction, "limit": str(limit)},
+        )
+        return [
+            e for e in resp.json().get("chunk", [])
+            if e.get("type") == "m.room.message"
+        ]
+
+    def register_user(
+        self,
+        username: str,
+        password: str,
+        admin: bool = False,
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)